### PR TITLE
showIf/Else with simple conditions

### PR DESCRIPTION
**Pull request by ddworak**

### CODE REVIEW

File: core/.js/src/main/scala/io/udash/bindings/Bindings.scala [hunk 1]
```diff
@@ -411,21 +426,19 @@ object Bindings extends Bindings {
       * If callback returns true, other listeners which are queued will not be invoked.
       * If callback returns false, next callback in the queue will be invoked.
       */
-    def :+=[T <: Event](callback: (T) => Boolean): Modifier[Element] = {
+    def :+=[T <: Event](callback: T => Boolean): Modifier[Element] = {
       AttrPair(attr, callback, new AttrValue[Element, (T) => Boolean] {
```
Comment (bgrochal) [inline core/.js/src/main/scala/io/udash/bindings/Bindings.scala]: `(T)` has unnecessary braces (in general, `AttrValue` might be implemented with single method)

File: core/.js/src/main/scala/io/udash/bindings/Bindings.scala [hunk 2]
```diff
@@ -411,21 +426,19 @@ object Bindings extends Bindings {
...
-        override def apply(el: Element, attr: Attr, callback: (T) => Boolean): Unit = {
+        override def apply(el: Element, attr: Attr, callback: T => Boolean): Unit = {
           val dyn: js.Dynamic = el.asInstanceOf[js.Dynamic]
           val existingCallbacks: js.Function1[T, Boolean] = dyn.selectDynamic(attr.name).asInstanceOf[js.Function1[T, Boolean]]
           if (existingCallbacks == null) {
-            dyn.updateDynamic(attr.name)((e: T) => {
-              if (callback(e)) e.preventDefault()
-            })
+            dyn.updateDynamic(attr.name)((e: T) => if (callback(e)) e.preventDefault())
```
Comment (bgrochal) [inline core/.js/src/main/scala/io/udash/bindings/Bindings.scala]: Could be:
```scala
dyn.updateDynamic(attr.name) { e: T => if (callback(e)) e.preventDefault() }
```
similarly to the `else` part.

File: core/.js/src/main/scala/io/udash/bindings/Bindings.scala [hunk 3]
```diff
@@ -465,7 +481,7 @@ object Bindings extends Bindings {
       }
   }
 
-  class AttrPairOps(private val attr: scalatags.generic.AttrPair[Element, _]) extends AnyVal { outer =>
+  final class AttrPairOps(private val attr: scalatags.generic.AttrPair[Element, _]) extends AnyVal { outer =>
```
Comment (bgrochal) [inline core/.js/src/main/scala/io/udash/bindings/Bindings.scala]: Renaming to `outer` is not necessary now.

File: core/.js/src/main/scala/io/udash/bindings/Bindings.scala [hunk 4]
```diff
@@ -134,21 +156,21 @@ trait Bindings {
     *
     * For example:
     * <pre>
-    *   produceWithNested(property) { case (data, nested) =>
```
[the remark this answers is not in the document]
Comment (ddworak) [reply core/.js/src/main/scala/io/udash/bindings/Bindings.scala]: IntelliJ does not consider `<pre>` as formatted Scala code (so it reformats). Added `{{{`

File: core/.js/src/main/scala/io/udash/bindings/Bindings.scala (hunk 1 above)
Comment (ddworak) [reply core/.js/src/main/scala/io/udash/bindings/Bindings.scala]: It can't, this cross-compiles to 2.11

**Pull request merged**